main.tf

```diff
@@ -9,6 +9,9 @@ locals {
   lacework_domain  = length(var.global_module_reference.lacework_domain) > 0 ? var.global_module_reference.lacework_domain : var.lacework_domain
   lacework_account = length(var.global_module_reference.lacework_account) > 0 ? var.global_module_reference.lacework_account : (length(var.lacework_account) > 0 ? var.lacework_account : trimsuffix(data.lacework_user_profile.current.url, ".${local.lacework_domain}"))
 
+  network_name        = length(var.global_module_reference.network_name) > 0 ? var.global_module_reference.network_name : var.network_name
```

Comment:
If global module reference is not present i.e this is the global module we get the network name from `var.network_name`
I do not see this being provided in any of the `main.tf` in the examples. Should we derive this using the `suffix` and `prefix` as well

Reply:
The `network_name` variable simply has a default of `lacework-awls` as defined in the variables.tf file:
```
variable "network_name" {
  type        = string
  default     = "lacework-awls"
  description = "The name of the network to create for agentless workload scanning resources."
}
```
It should be necessary for it to be unique, so I don't know that we need to add the random suffix, but we certainly can.